### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by turtley12** [saucedacity/saucedacity#75] [General]
I'm curious if there are any plans for how to do documentation.

As far as I know Tenacity never had much documentation, and since it was similar enough to audacity, they suggested using the audacity documentation.

As Saucedacity/Tenacity gets further and further from Audacity, having it's own documentation would likely be a good idea.

This also then brings up the question of how the documentation would work, Readthedocs, GitHub pages, another documentation system, etc.

Personally, I like the idea of using a Readthedocs look-alike theme in GitHub pages. This way we'd keep as much as possible in GitHub, and also have the familiar Readthedocs look. There are certainly other options too, and I'm looking forward to seeing what other people think.

Apologies if this has already been decided/discussed.

Turtley12

**Comment by generic-pers0n** [comment]
I have not discussed documentation at all...

I believe we should set this up as we start to make releases. I'll experiment with building the Audacity manual and see how that works.
